docs/usage/configuration-options.md

```diff
@@ -802,9 +802,24 @@ For the full list of available managers, see the [Supported Managers](https://do
 
 ## encrypted
 
-Use this to encrypt secrets in a way which can be stored in repository configs.
+Use this to encrypt secrets in a way which can be stored in repository configs. It is possible to encrypt secrets using either a HTML page, or CLI.
 
-See [Private module support](https://docs.renovatebot.com/getting-started/private-packages) for details on how this is used to encrypt npm tokens.
+To encrypt secrets for the hosted Mend Renovate app for github.com using a HTML page, go to [app.renovatebot.com/encrypt](https://app.renovatebot.com/encrypt) and complete the form. Alternatively, this form may be downloaded and edited for self-hosted users to use their own PGP public key.
+
+You can also encrypt secrets using the CLI, using the `gpg` command line tool plus some help from `curl` and `jq`. Here is an example:
+
+```
+curl https://app.renovatebot.com/renovate.pgp --output renovate.pgp
+echo -n '{"o":"your-organization", "r":"your-repository (optional)", "v":"your-secret-value"}' | jq . -c | gpg --encrypt -a --recipient-file renovate.pgp | grep -v '^----' | tr -d '\n'
+```
+
+The above script does this:
+
+- Uses `curl` to download the Mend Renovate hosted app's public key
+- Echos a JSON object into `jq` for validation and compacting
+- Uses `gpg` to encrypt the contents
+
+The `jq` step is optional, you could leave it out if you wish. Its primary value is validating that the string you input to `gpg` is valid JSON, and compact.
```

Comment:
```suggestion
The above script uses:

- `curl` to download the Mend Renovate hosted app's public key
- `echo` to echo a JSON object into `jq`
- `jq` to validate the JSON and then compact it
- `gpg` to encrypt the contents
- `grep` to ...
- `tr` to ...

The `jq` step is optional, you can leave it out if you wish.
Its primary value is validating that the string you echo to `gpg` is valid JSON, and compact.
```

Before you apply this suggestion, fill out the `...` parts to explain what that part of the command does. 😉